aligner/cli.py

```diff
@@ -21,6 +21,64 @@ def complete_path():
     return []
 
 
+@app.command()
```

Comment:
Please add `short_help="..."` and `help="""..."""` here, possibly just copied from what you added to `everyvoice/cli.py` where you copy these commands.

We should look for a way to place the short help and long help messages in the submodule, and import them into the EV CLI, instead of having to repeat them.

Reply:
PS: Don't forget to add `rich_markup_mode="markdown",` to the `a = typer.Typer()` constructor if you use markdown in the long help here.

Reply:
And `ctc-segmenter -h` could really use the long help you wrote for `everyvoice segment -h`, which explains in more details what align and extract do.